### PR TITLE
Update to Swift 3 syntax for Xcode 8.0 public release

### PR DESCRIPTION
**Pull request by ephraimkunz**

### CODE REVIEW

File: Cosmos/CosmosRating.swift [hunk 1]
```diff
@@ -18,7 +18,7 @@ struct CosmosRating {
   - returns: Decimal value between 0 and 1 describing the star fill level. 1 is a fully filled star. 0 is an empty star. 0.5 is a half-star.
   
   */
-  static func starFillLevel(ratingRemainder: Double, fillMode: StarFillMode) -> Double {
+  static func starFillLevel(_ ratingRemainder: Double, fillMode: StarFillMode) -> Double {
```
Comment (evgenyneu) [inline Cosmos/CosmosRating.swift]: Why this is needed?

File: Distrib/CosmosDistrib.swift [hunk 2]
```diff
@@ -910,7 +910,7 @@ Shows: ★★★★☆ (132)
   }
   
   /// Change view settings for faster drawing
-  private func improvePerformance() {
+  fileprivate func improvePerformance() {
```
Comment (evgenyneu) [inline Distrib/CosmosDistrib.swift]: Why do we need to use `fileprivate`?

File: Cosmos/CosmosView.swift [hunk 3]
```diff
@@ -28,7 +28,7 @@ Shows: ★★★★☆ (132)
   }
   
   /// Currently shown text. Set it to nil to display just the stars without text.
-  @IBInspectable public var text: String? {
+  @IBInspectable open var text: String? {
```
Comment (evgenyneu) [inline Cosmos/CosmosView.swift]: Why we need to use `open` instead of `public` in this class?

**Comment by ephraimkunz** [comment]
It looks like it is working with no changes now. For some reason, when opening my project in Swift 2 in Xcode 8, it recommended changes to update the library to Swift 3. Thanks for a great library!

**Comment by evgenyneu** [comment]
No worries, all the best.

**Pull request closed without merging**